app/models.py

```diff
@@ -50,6 +50,11 @@
 TEMPLATE_PROCESS_TYPE = [NORMAL, PRIORITY]
 
 
+SMS_AUTH_TYPE = 'sms_auth'
```

Comment:
could just use sms and email. Rather than put '_auth' on. the column is called auth_type.